### DISCUSSION TITLE
Update v0.19.3 and Certificate expiring

### DISCUSSION THREAD

**Discussion by ederpardeiro** [amidaware/tacticalrmm#2030] [Ideas]
Hi guys, how are you?

How could I fix this two issues?

![image](https://github.com/user-attachments/assets/3b3527c9-950d-4981-bfb7-334e98670b99)

**Comment by dinger1986** [comment]
Following these instructions will resolve both https://docs.tacticalrmm.com/update_server/#keeping-your-lets-encrypt-certificate-up-to-date

**Reply by ederpardeiro** [reply]
Thank you very much!!!

All it's working perfecly now!!!

**Reply by pdifeo** [reply]
For those who are not aware of it.

certbot has the possibility of issuing/renewing certificates via plugins that use the DNS APIs of various providers.

These are the DNS plugins for Ubuntu 22.04

```
python3-certbot-dns-cloudflare/jammy 1.18.0-1 all
python3-certbot-dns-digitalocean/jammy 1.18.0-1 all
python3-certbot-dns-dnsimple/jammy 1.18.0-1 all
python3-certbot-dns-gandi/jammy 1.3.2-1 all
python3-certbot-dns-gehirn/jammy 1.18.0-1 all
python3-certbot-dns-google/jammy 1.18.0-1 all
python3-certbot-dns-linode/jammy 1.7.0-1 all
python3-certbot-dns-ovh/jammy,now 1.3.0-1 all [installed]
python3-certbot-dns-rfc2136/jammy 1.21.0-1 all
python3-certbot-dns-route53/jammy 1.3.0-1 all
python3-certbot-dns-sakuracloud/jammy 1.3.0-1 all
python3-certbot-dns-standalone/jammy 1.0.3-2 all
```

they are all fairly well documented. 

I use python3-certbot-dns-ovh and have automated the renewal. This way I don't have to go crazy every 3 months.

Regards
